lib/SlackBot.js

```diff
@@ -4,6 +4,7 @@ var express = require('express');
 var bodyParser = require('body-parser');
 var querystring = require('querystring');
 var async = require('async');
+var Cryptr = require('cryptr');
```

Comment:
@anonrig In my case, I'm using hosted DB solution. Compromised DB will have a severe impact as I have data for few hundred teams (And thousands of users). I believe I can protect the encryption key (on app side) better than the database storage/access. 

I know it's a precaution so wanted to keep it optional.